package.json

```diff
@@ -29,12 +31,11 @@
   },
   "devDependencies": {
     "babel-cli": "^6.6.5",
-    "babel-core": "^6.7.2",
     "babel-preset-es2015": "^6.6.0",
+    "babel-register": "^6.7.2",
     "chai": "^3.5.0",
     "eslint": "2.3.0",
     "eslint-config-airbnb": "6.1.0",
-    "eslint-plugin-react": "^4.2.3",
```

Comment:
Everything seems to work but I'm getting the following warning when installing with node 5 and npm 3:
<img width="647" alt="screen shot 2016-04-03 at 1 35 01 pm" src="https://cloud.githubusercontent.com/assets/1198196/14233819/f7f35204-f9a0-11e5-8560-4c2cc3a97d57.png">